>From f59851fb956642e0a43e296fb04bfab5577ae534 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Wed, 4 Sep 2013 09:35:22 +0300
Subject: [PATCH 1/3] In debian-ispell.el load setup after ispell, not at
 Emacs start up
Organization: Private
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 support/emacsen/debian-ispell.el |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/support/emacsen/debian-ispell.el b/support/emacsen/debian-ispell.el
index 4a57a0b..b5e9a59 100644
--- a/support/emacsen/debian-ispell.el
+++ b/support/emacsen/debian-ispell.el
@@ -251,10 +251,10 @@ To be run at `after-init-hook' or at any time if FORCE is given."
 	))))
 
 ;; Make sure updated Debian menu is available after emacs is started
-(add-hook 'after-init-hook 'debian-ispell-set-startup-menu)
+;; (add-hook 'after-init-hook 'debian-ispell-set-startup-menu)
 
 ;; Make sure updated Debian menu is not overriden by ispell.el one
-(eval-after-load "ispell" '(debian-ispell-set-startup-menu))
+(eval-after-load "ispell" '(debian-ispell-set-startup-menu 'force))
 
 ;;; -----------------------------------------------------------------------
 ;;;  Guess default ispell dictionary under emacs and make ispell.el use it
-- 
1.7.10.4

